cli/src/target.ts

```diff
@@ -62,7 +63,10 @@ export default class Target {
 
   async clean() {
     // Remove the directory associated with this target.
-    await remove(path.resolve(this.crate.root, 'target', this.subdirectory));
+    const absolutePathSubdir = path.resolve(this.crate.root, 'target', this.subdirectory);
+    if (existsSync(absolutePathSubdir)) {
+      await unlink(absolutePathSubdir);
```

Comment:
`unlink` is for files or symlinks not directories, should we be using rimraf instead?

Reply:
Ah good catch. It seems like node has a [rmdir](https://nodejs.org/api/fs.html#fs_fs_rmdirsync_path_options) API. What are your thoughts on that?

Reply:
If a user is running `neon clean` there's probably still files in the `target` folder, so we need a recursive rmdir. The `recursive` variant of `fs.rmdir` is only supported in node 12 and up, so I think we should use the rimraf module until we drop node 10

Reply:
Makes sense. I'll update this PR to reflect that